### PR TITLE
Bump version: 4.2.2 → 4.2.3: Use dict instead of MutableMapping

### PR DESCRIPTION
**Pull request by lgruen**
Should not result in any functional change.

### CODE REVIEW

**Review by MattWellie — APPROVED** [review]

File: cpg_utils/config.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-from typing import Any, MutableMapping, Optional
+from typing import Optional
 from cloudpathlib import AnyPath
 import toml
 
 # We use these globals for lazy initialization, but pylint doesn't like that.
 # pylint: disable=global-statement, invalid-name
 _config_path = os.getenv('CPG_CONFIG_PATH')  # See set_config_path.
-_config: Optional[MutableMapping[str, Any]] = None  # Cached config, initialized lazily.
+_config: Optional[dict] = None  # Cached config, initialized lazily.
```
Comment (illusional) [inline cpg_utils/config.py:11]: I prefer `Dict` from typing, then you can annotate it like `Dict[str, Any]`, but this is fine too

**Review by illusional — APPROVED** [review]
Only nitpick about which `dict / Dict` to type annotate with

File: cpg_utils/config.py (hunk 1 above)
Comment (vladsavelyev) [reply cpg_utils/config.py:11]: @illusional you can use `dict[str, Any]`

**Review by vladsavelyev — APPROVED** [review]

File: cpg_utils/config.py (hunk 1 above)
Comment (illusional) [reply cpg_utils/config.py:11]: Ah that's a Python 3.10 thing I see

**Pull request merged**